packages/core/dao/UserManagerDAO.js

```diff
@@ -34,73 +17,27 @@ export default class UserManagerDAO extends AbstractContractDAO {
 
   connect (web3, options) {
     super.connect(web3, options)
-
-    this.CBEUpdateEmitter = this.contract.events.CBEUpdate({})
-      .on('data', this.handleCBEUpdateData.bind(this))
-      .on('error', this.handleCBEUpdateError.bind(this))
-
-    this.NewUserRegisteredEmitter = this.contract.events.NewUserRegistered({})
-      .on('data', this.handleNewUserRegisteredData.bind(this))
-      .on('error', this.handleNewUserRegisteredError.bind(this))
   }
 
   disconnect () {
     if (this.isConnected) {
-      this.CBEUpdateEmitter.removeAllListeners()
-      this.NewUserRegisteredEmitter.removeAllListeners()
       this.contract = null
       this.history = null
       this.web3 = null
     }
   }
 
-  handleCBEUpdateData = (data) => {
-    if (!data.event) {
-      return
-    }
-
-    this.emit('CBEUpdate', data)
-  }
-
-  handleCBEUpdateError = (data) => {
-    this.emit('CBEUpdate_error', data)
-  }
-
-  handleNewUserRegisteredData = (data) => {
-    if (!data.event) {
-      return
-    }
-
-    this.emit('NewUserRegistered', data)
-  }
-
-  handleNewUserRegisteredError = (data) => {
-    this.emit('NewUserRegistered_error', data)
-  }
-
   isCBE (account): Promise<boolean> {
     return this.contract.methods.getCBE(account).call()
   }
 
-  async usersTotal () {
-    const userCount = await this.contract.methods.userCount().call()
-    return userCount - 1
-  }
-
   getMemberId (account) {
     return this.contract.methods.getMemberId(account).call()
   }
 
-  getSignsRequired () {
-    return this.contract.methods.required().call()
-  }
-
-  getAdminCount () {
-    return this.contract.methods.adminCount().call()
-  }
-
   async getCBEList (): Immutable.Map<CBEModel> {
-    const [ addresses, hashes ] = await this.contract.methods.getCBEMembers().call()
+    const result = await this.contract.methods.getCBEMembers().call()
+    const [addresses, hashes] = Object.values(result)
```

Comment:
What if result will be undefined? Object.values will throw the exception `TypeError: Cannot convert undefined or null to object`.